stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/input/MessageInputView.kt

```diff
@@ -130,8 +136,17 @@ public class MessageInputView : ConstraintLayout {
         this.sendMessageHandler = handler
     }
 
+    @Deprecated(
+        "Use setUserLookupHandler instead of manually passing the list of users",
+        ReplaceWith(
+            "setUserLookupHandler(DefaultUserLookupHandler(members.map(Member::user)))",
+            "io.getstream.chat.android.ui.message.input.MessageInputView.DefaultUserLookupHandler",
```

Comment:
`DefaultUserLookupHandler` is an internal class. 
We shouldn't recommend it as a replacement when a method is deprecated.
What do you think about to make public this class? :thinking:

Reply:
Yes, we can expose it. Fixed